vendor/licenses/grammar/vscode-prisma.txt

```diff
@@ -0,0 +1,6 @@
+---
+type: grammar
+name: vscode-prisma
+version: ef6481235feb87fb7c0888745bbbb6ed6861175e
+license: apache-2.0
+---
```

Comment:
The license text appears to be missing here. Did you use the add-grammar script?

Reply:
Just adjusted it.

Reply:
🤔 the actual license text should have been pulled in when running `add-grammar`. I've just tested locally and it works for me:

```console
$ script/add-grammar https://github.com/prisma/vscode-prisma 
Checking docker is installed and running
$ docker ps
Registering new submodule: vendor/grammars/vscode-prisma
$ git submodule add -f https://github.com/prisma/vscode-prisma vendor/grammars/vscode-prisma
$ script/grammar-compiler add vendor/grammars/vscode-prisma
Confirming license
$ script/licensed --module /Users/lildude/github/linguist/vendor/grammars/vscode-prisma
Updating grammar documentation in vendor/README.md
$ bundle exec rake samples
$ script/sort-submodules
$ script/list-grammars
$ head vendor/licenses/grammar/vscode-prisma.txt
---
type: grammar
name: vscode-prisma
version: e90368f8d15ed30fe95f3f83e252132a3091f845
license: apache-2.0
---
                                 Apache License
                           Version 2.0, January 2004
                        http://www.apache.org/licenses/
$
```
Can you please try again or manually add the license text to this file.